R/tidyverse.R

```diff
@@ -286,9 +286,11 @@ use_tidy_thanks <- function(repo_spec = github_repo_spec(),
     return(invisible())
   }
 
+  res_POSIXct <- as.POSIXct(pluck_chr(res, "created_at"))
+  res <- res[res_POSIXct >= from_timestamp]
+
```

Comment:
We have to do something about the fact that `res_POSIXct` may no longer be synced with `res` before the next block, where it gets used again.

Reply:
You are right, made a change which should fix this and hopefully clean it up a little.